Assets/MixedRealityToolkit/Providers/UnityInput/UnityTouchController.cs

```diff
@@ -14,6 +14,23 @@ public class UnityTouchController : BaseController
         public UnityTouchController(TrackingState trackingState, Handedness controllerHandedness, IMixedRealityInputSource inputSource = null, MixedRealityInteractionMapping[] interactions = null)
                 : base(trackingState, controllerHandedness, inputSource, interactions)
         {
+            if (InputSystem?.InputSystemProfile.GesturesProfile != null)
```

Comment:
Why did this move out of `SetupDefaultInteractions`? Is that not being called or not being called reliably?